incident_test.go

```diff
@@ -649,7 +649,7 @@ func TestIncident_ManageAlerts(t *testing.T) {
 			},
 		},
 	}
-	res, _, err := client.ManageIncidentAlerts(incidentID, input)
+	res, _, err := client.ManageIncidentAlerts(from, incidentID, input)
```

Comment:
This doesn't seem to actually assert that the `from` value is making it through as an HTTP header. Could you update the handler above (on line `635`) to look for that header and the correct value, and fail the request/test otherwise?